### PR TITLE
Add csv logger to some examples (mnist & stoic)

### PR DESCRIPTION
**Pull request by smartdanny**

### CODE REVIEW

**Review by mosheraboh — APPROVED** [review]
Looks good,
One question inline.

File: examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py [hunk 1]
```diff
@@ -305,6 +307,7 @@ def run_train(paths: dict, train_params: dict):
         strategy=train_params["trainer.strategy"],
         devices=train_params["trainer.num_devices"],
         auto_select_gpus=True,
+        logger=lightning_csv_logger,
```
[the remark this answers is not in the document]
Comment (smartdanny) [reply examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py:310]: If we use CSVLogger such as in these examples, then PL will not use tensorboard logger.

[the remark this answers is not in the document]
Comment (mosheraboh) [reply examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py:310]: Thanks,
So lets pass in the examples list of two loggers (csv and tensorboard)?

[the remark this answers is not in the document]
Comment (smartdanny) [reply examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py]: done :)

**Review by mosheraboh — APPROVED** [review]

**Pull request merged**